source/shared/cpp/ObjectModel/Carousel.cpp

```diff
@@ -62,6 +66,11 @@ Json::Value Carousel::SerializeToJsonValue() const
         root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Timer)] = m_timer.value_or(0);
     }
 
+    if (m_rtl.has_value())
+    {
+        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Rtl)] = m_rtl.value_or("");
```

Comment:
Why doesn't this default to true/false?

Reply:
it's because ios/android std lib.  value_or would return a value if it doesn't. so we have this check, has_value(), the default value shouldn't really matter because of the check.